plugins/arDominionPlugin/css/less/scaffolding.less

```diff
@@ -1668,6 +1668,51 @@ body.login #content {
   }
 }
 
+// Bootstrap 2 theme deprecation notice
+@media (max-width: 767px) {
+  #bs2-deprecation-message {
+    margin-left: -20px;
+    margin-right: -20px;
+  }
+}
+
+#bs2-deprecation-message {
+    text-align: center;
+    position: sticky;
+    top: 0;
+    z-index: 1;
+
+    .alert-banner {
+        margin: 0 auto;
+        display: flex;
+        justify-content: space-between;
+        border: 0;
+        border-radius: 0;
+    }
+
+    .bs2-deprecation-message-button {
+        font-size: 1.75rem;
+        right: 0;
+        top: 0;
+        transform: translateX(-100%);
+    }
+
+    #bs2-deprecation-message-content {
+        padding: 8px 0;
+        color: @red;
+        position: relative;
+        left: 50%;
+        transform: translateX(-50%);
+        margin-right: 65px;
+        font-size: 0.85rem;
+
+        a:not(.btn) {
+            text-shadow: none;
+            text-decoration: underline;
+        }
+    }
+}
```

Comment:
Some of this styling is just centering the content and aligning the close icon with the buttons from the privacy and update check notifications to be consistent, and some of it is to visually distinguish it from those two since there is a scenario where a user might see all three, and it made sense to make this appear more prominent than the other two. I could potentially combine some of the styles for all three of these under the same classes though and clean some of the duplicated code up (there's a little bit that's duplicated between the other two alerts already).

Good catch on the indentation though, it seems like my editor's indentation default was set to 4 instead of 2 spaces for CSS!